nodejs/overview/msteams-client.md

```diff
@@ -1,61 +1,111 @@
 
-www.npmjs.com/package/@microsoft/teams-js
+### Finding the library
 
-## Installing the library
+The Live share SDK is a JavaScript package comprised of three sub-packages ([Live Share](msteams-docs/msteams-platform/apps-in-teams-meetings/teams-live-share-capabilities.md), [Live Share media](msteams-docs/msteams-platform/apps-in-teams-meetings/teams-live-share-media-capabilities.md), and [Live Share canvas](msteams-docs/msteams-platform/apps-in-teams-meetings/teams-live-share-canvas.md)) which is published on npm and can be installed using npm or yarn.
 
-You can install the package using npm or yarn:
+#### Live share
 
-* `npm install --save @microsoft/teams-js`
-* `yarn add @microsoft/teams-js`
+To install the latest version of the Live share package to your application:
 
-## Using the library
+To install using npm
 
-If you are using any dependency loader or module bundler such as [RequireJS](http://requirejs.org/), [SystemJS](https://github.com/systemjs/systemjs), [browserify](http://browserify.org/), or [webpack](https://webpack.github.io/), you can use `import` syntax to import specific modules. For example:
+```bash
+npm install @microsoft/live-share@next --save
+```
+OR
+
+To install using yarn
+```bash
+yarn add @microsoft/live-share@next
+```
+
+#### Live Share media
+To install the latest version of the Live Share media package to your application:
+
+To install using npm
+
+```bash
+npm install @microsoft/live-share@next --save
+npm install @microsoft/live-share-media@next --save
+```
+OR
+
+To install using yarn
+
+```bash
+yarn add @microsoft/live-share@next
+yarn add @microsoft/live-share-media@next
+```
 
-```typescript
-import * as microsoftTeams from "@microsoft/teams-js";
+#### Live Share canvas
+
+To install the latest version of the Live Share media package to your application:
+
+To install using npm
+
+```bash
+npm install @microsoft/live-share@next --save
+npm install @microsoft/live-share-canvas@next --save
+```
+OR
+
+To install using yarn
+
+```bash
+yarn add @microsoft/live-share@next
+yarn add @microsoft/live-share-canvas@next
 ```
 
-You can also reference the entire library in html pages using a script tag.  There are three ways to do this:
+## Microsoft Teams JavaScript client library
+
+The Microsoft Teams JavaScript client library version `2.x.x` is enabled [run apps across Microsoft 365](/microsoftteams/platform/m365-apps/overview). The TeamsJS SDK is a superset of TeamsJS `1.x` versions: and supports existing Teams app functionality while adding the ability to host Teams apps in Outlook and Microsoft 365. Refer to the [Teams JavaScript client library](/microsoftteams/platform/tabs/how-to/using-teams-client-library) overview for guidance on building Microsoft 365-enabled apps.
```

Comment:
The "is enabled" text is still odd to me. I'd suggest rewording. This also just differentiates between v2 and v1, it doesn't really introduce the client library. Maybe a couple sentences before discussing the versions.